packages/fiori-elements-writer/package.json

```
@@ -0,0 +1,45 @@
+{
+    "name": "@sap-ux/fiori-elements-writer",
+    "version": "0.0.1",
+    "license": "Apache-2.0",
+    "main": "dist/index.js",
+    "scripts": {
+        "build": "rimraf dist && tsc",
+        "clean": "rimraf dist test/test-output coverage",
+        "watch": "tsc --watch",
+        "lint": "eslint . --ext .ts",
+        "test": "jest --ci --forceExit --detectOpenHandles --colors",
+        "test-u": "jest --ci --forceExit --detectOpenHandles --colors -u",
+        "test:local": "node -r ts-node/register test/index.ts",
+        "link": "pnpm link --global",
+        "unlink": "pnpm unlink --global"
+    },
+    "files": [
+        "LICENSE",
+        "dist",
+        "templates"
+    ],
+    "dependencies": {
+        "@sap-ux/odata-service-writer": "workspace:*",
+        "@sap-ux/ui5-application-writer": "workspace:*",
+        "@sap-ux/ui5-config": "workspace:*",
+        "ejs": "3.1.6",
+        "i18next": "20.3.2",
+        "lodash": "4.17.21",
+        "mem-fs": "2.1.0",
+        "mem-fs-editor": "9.0.0",
+        "read-pkg-up": "7.0.1",
+        "semver": "7.3.5"
+    },
+    "devDependencies": {
+        "@types/ejs": "3.1.0",
+        "@types/fs-extra": "9.0.13",
+        "@types/lodash": "4.14.176",
+        "@types/mem-fs-editor": "7.0.1",
+        "fs-extra": "10.0.0"
+    },
+    "engines": {
+        "pnpm": ">=6.0.2",
```

```suggestion
        "pnpm": ">=6.26.1",
```
To be consistent with the other packages in the `main` branch.